practice/validator/site.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+// 1. Input
+/**
+ * Gathers what has been typed in the 'name' field for each event.
+ */
+function getUserName() {
+  return $('#name').val();
+}
+/**
+ * Gathers what has been typed in the 'DOB' field for each event.
+ */
+function getUserDOB() {
+  return $('#dob').val();
+}
+/**
+ * Gathers what has been typed in the 'Phone' field for each event.
+ */
+function getUserPhone() {
+  return $('#phone').val();
+}
+
+// 2. Transform Functions
+// No transform.
+
+// 3. Create
+var NAME_MATCH = /\w+\s\w+/;
+var DOB_MATCH = /\d\d\d\d-\d\d-\d\d/;
+var PHONE_MATCH = /\d\d\d-\d\d\d-\d\d\d\d/;
+/**
+ * Selects the 'name' input box.
+ */
+function getNameBox() {
+  return $('#name');
+}
+/**
+ * Selects the 'dob' input box.
+ */
+function getDOBBox() {
+  return $('#dob');
+}
+/**
+ * Selects the 'phone' input box.
+ */
+function getPhoneBox() {
+  return $('#phone');
+}
```

Comment:
Since your id's are very descriptive (#name, #dob, #phone), I would not have any trouble understanding your code if you eliminated these functions and just used the $('#...') where the call was made.